app/views/bulkrax/importers/show.html.erb

```diff
@@ -80,120 +80,9 @@
       </ul>
       <!-- Tab panes -->
       <div class="tab-content outline">
-        <div role="tabpanel" class="tab-pane active bulkrax-nav-tab-table-left-align" id="work-entries">
```

Comment:
moved the entry tabs into partials.

Reply:
thanks for doing so, although both worked this looks so much better and less overwhelming to read :)